charts/kube-state-metrics/Chart.yaml

```diff
@@ -6,7 +6,7 @@ keywords:
 - monitoring
 - prometheus
 - kubernetes
-version: 2.9.3
+version: 2.10.1
```

Comment:
can we go for 2.10.0 (since it adds a new feature) instead?

Reply:
agreed :)